### ISSUE TITLE
Ruffle web emulator fails on Chrome 86 on Android 11 in Portrait mode - 'tall' files problem?

### ISSUE THREAD

**Issue by aswiseman** [ruffle-rs/ruffle#1775]
This URL: [https://www.625.uk.com/tv_logos/flash/abc_bb.asp?wasm=1](https://www.625.uk.com/tv_logos/flash/abc_bb.asp?wasm=1) is running the Ruffle emulator web assembly.

When I visit on my Pixel 3a XL, which is running Chrome 86 on Android 11, the page loads, I get the big orange triangle, but then nothing. I can then get an error message (see below)(although fiddling around trying to get an error message may have caused the error rather than anything else?)

(I wasn't able to copy and paste the error - I think the right-click Ruffle menu gets in teh way - so I had to make screenshots and then use Google Photos text capture - so there may be some transcription errors)

Curiously, if I'm in split-screen mode (that is Chrome is sharing the screen with another app), the Flash animation works fine. And if I'm in landscape mode, it also works fine. But in straight portrait mode, just a black screen and then eventually the error message.

It's also fine running Microsoft Edge browser and Firefox on the same phone.

I've tested this on a Pixel 3a (non-XL) and have seen the same issue.

Which make me wonder if there's a problem rendering 'tall' Flash files?

# Error Info
Error name: Error
Error message: panicked at 'called
`Result:: unwrap ()' on an `Err` value:
"WebGL: Error in
renderbuffer_storage_multisample (color):
1281"', render/webgl/src/lib.rs:739:35
Error stack:
```
Error: panicked at 'called
`Result:: unwrap() on an `Err` value:
"WebGL: Error in
renderbuffer_storage_multisample (color):
1281"', render/webgl/src/lib.rs:739:35
	at Object.$
(https://www.625.uk.com/tv_logos/flash/ruf fle/core.ruffle.7f6d216454b6a4a07936.js:1: 20278)
	at _wbg_new_df6e6ab7a65c4c4d
(https://www.625.uk.com/tv_logos/flash/ruf fle/ruffle.js:1:31106)
	at https://www.625.uk.com/tv_logos/flash/ruffle/85f5e4a29d83ee43a391.module.wasm:wasm-function[1201]:0x22757d
	at
https://www.625.uk.com/tv_logos/flash/ruff le/85f5e4a29d83ee43a391.module.wasm:wasm-function[1358] : 0x2436eb
	at
https://www.625.uk.com/tv_logos/flash/ruff le/85f5e4a29d83ee43a391.module.wasm:wasm-function[3589] : 0x2eee47
	at
https://www.625.uk.com/tv_logos/flash/ruffle/85f5e4a29d83ee43a391.module.wasm:wasm-
function [2813] : 0x2d15fd
	at
https://www.625.uk.com/tv_logos/flash/ruffle/85f5e4a29d83ee43a391.module.wasm:wasm-function [3884] : 0x2f3a65
	at
https://www.625.uk.com/tv_logos/flash/ruffle/85f5e4a29d83ee43a391.module.wasm:wasm-function[3913] : 0x2f3f9a
	at
https://www.625.uk.com/tv_logos/flash/ruffle/85f5e4a29d83ee43a391.module.wasm:wasm- function[3789] : 0x2f2642
	at
https://www.625.uk.com/tv_logos/flash/ruffle/85f5e4a29d83ee43a391.module.wasm:wasm-function[3914] : 0x2f3fdo
```

# Player Info
Allows script access: true
Player type: Object
SWF URL: abc_bb.swf
Param movie: abc_bb.swf
Param quality: high
Param bgcolor: #000000
Attribute 0: [object Attr]
Attribute 1: [object Attr]
Attribute 2: [object Attr]
Attribute 3: [object Attr]
Attribute 4: [object Attr]

# Page Info
Page URL:
https://www.625.uk.com/tv_logos/flash/abc_bb.asp?wasm=1

# Browser Info
Useragent: Mozilla/5.0 (Linux; Android 11; Pixel 3a XL) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/86.0.4240.198 Mobile Safari/537.36
OS: Linux armv81

# Ruffle Info
Ruffle version: 0.1.0
Ruffle source: undefined

**Comment by Toad06** [comment]
I think this may have been fixed by #1767. Could you try updating your build to the [latest nightly](https://github.com/ruffle-rs/ruffle/tags) and let us know if it's still an issue? Thanks in advance!

**Comment by aswiseman** [comment]
> I think this may have been fixed by #1767. Could you try updating your build to the [latest nightly](https://github.com/ruffle-rs/ruffle/tags) and let us know if it's still an issue? Thanks in advance!

I've just uploaded the nightly web assembly from 2020-12-02. You're absolutely correct - the problem is solved. 
(To confirm: my Pixel 3a XL is now happily running Flash content in portrait mode.)

Thank you!
